ee/src/main/java/org/jboss/as/ee/subsystem/ManagedScheduledExecutorServiceResourceDefinition.java

```diff
@@ -46,6 +48,12 @@
  */
 public class ManagedScheduledExecutorServiceResourceDefinition extends SimpleResourceDefinition {
 
+    /**
+     * the resource definition's dynamic runtime capability
+     */
+    public static final RuntimeCapability<Void> CAPABILITY = RuntimeCapability.Builder.of("org.wildfly.ee.concurrent.scheduledexecutor", true, ManagedScheduledExecutorServiceAdapter.class)
```

Comment:
For readability, we generally use dashes when converting camel case names to capability names, e.g. "org.wildfly.ee.concurrent.scheduled-executor"

Reply:
done